libs/features/personalization/personalization.js

```diff
@@ -393,6 +397,10 @@ const getVariantInfo = (line, variantNames, variants) => {
         val: normalizePath(line[vn]),
         action,
       });
+    } else if ((action in COMMANDS || action in CREATE_CMDS)
+        && variantInfo.selectorType === GLOBAL_NAV_SELECTOR) {
```

Comment:
super nit: swap the && so that selectorType is first so it's a bit more obvious it's for gnav
```suggestion
    } else if (variantInfo.selectorType === GLOBAL_NAV_SELECTOR
        && (action in COMMANDS || action in CREATE_CMDS)) {
```